packages/vite-plugin-svelte/src/utils/resolve.ts

```diff
@@ -14,7 +14,13 @@ export function resolveViaPackageJsonSvelte(importee: string, importer?: string)
 }
 
 function isBareImport(importee: string): boolean {
-	if (!importee || importee[0] === '.' || importee[0] === '\0' || path.isAbsolute(importee)) {
+	if (
+		!importee ||
+		importee[0] === '.' ||
+		importee[0] === '\0' ||
+		importee.includes(':') ||
```

Comment:
Should we just do `.startsWith('node:')`? What else is this trying to catch?

Reply:
According to [node](https://nodejs.org/api/esm.html#urls), this would also catch `file:` and `data:`. And sometimes virtual imports uses `:` too, so we can catch it here.

Reply:
`:` is not allowed in paths in Windows, but it is on Linux (and presumably other UNIXes), which is making me concerned about handling this broadly like this. Would `/^[a-z]+:/.test(importee)` work?

Reply:
I think it should still be fine since we want to make sure `importee` is a node_modules package. So it has to be something like `foobar` or `@foo/bar` for this function to return true. And package names can't contain `:`. Unless I'm missing something 🤔

Reply:
Ah, I see, yep. Since this is only concerned with whether this is a bare import of the top level of a `node_modules` package, then even something like `foo/bar:baz` being caught by this would be fine, since it would be caught later on anyway.